frontend/model/contracts/shared/getters/group.js

```diff
@@ -0,0 +1,256 @@
+import { INVITE_STATUS } from '~/shared/domains/chelonia/constants.js'
+import {
+  INVITE_INITIAL_CREATOR,
+  MAX_SAVED_PERIODS,
+  PROFILE_STATUS,
+  PROPOSAL_GENERIC
+} from '../constants.js'
+import currencies from '../currencies.js'
+import { createPaymentInfo, paymentHashesFromPaymentPeriod } from '../functions.js'
+import { PAYMENT_COMPLETED } from '../payments/index.js'
+import { addTimeToDate, dateFromPeriodStamp, dateIsWithinPeriod, dateToPeriodStamp, periodStampsForDate } from '../time.js'
+
+export default ({
+  currentGroupLastLoggedIn () {
+    return {}
+  },
```

Comment:
This is not a shared getter, note that its definition is different in state.js:

```js
  currentGroupLastLoggedIn (state) {
    return state.lastLoggedIn[state.currentGroupId] || {}
  },
```

So it belongs directly in the contract.

Reply:
fixed